tifeatures/layer.py

```diff
@@ -261,6 +261,19 @@ def _datetime_filter_to_sql(self, interval: List[str], dt_name: str):
                     logic.V(dt_name) < logic.S(pg_funcs.cast(end, "timestamptz")),
                 )
 
+    def _sortby(self, sortby: Optional[str]):
+        sortby = sortby.strip()
+        if sortby is None:
+            sortby = self.id_column
+        if sortby.startswith("-"):
+            sortby = sortby.replace("-", "")
```

Comment:
is it possible that a `column` has a `-` in its name ?

Reply:
yep, good point

Reply:
last push using regexp should deal with this.